volplugin/handlers.go

```diff
@@ -226,6 +227,7 @@ func mount(master, host string, ttl int) func(http.ResponseWriter, *http.Request
 
 		actualSize, err := volConfig.Options.ActualSize()
 		if err != nil {
+			removeStopChan(uc.Request.Name)
```

Comment:
oh, sorry I should have been more clear, I meant do something like below which will ensure we only cleanup in case of error. But current way works too, it's just seemed that we have to remember adding this call before every return :
```
defer func() {
    if err != nil {
        removeStopChan(uc.Request.Name)
    }
}()
```